src/amo/components/Category/index.js

```diff
@@ -250,8 +253,20 @@ export class CategoryBase extends React.Component {
 
     const { html } = this.contentForType(addonType);
 
+    const isAddonTheme = isTheme(addonType);
```

Comment:
This only seems to be used once, below, so it's unnecessary. You could just do: `if (isTheme(addonType))` below instead of `if (isAddonTheme)`.

Reply:
I am using this 2x - on line 260 and line 267. is that ok then?

Reply:
now I using it many times :)

Reply:
So you are! 👍